source/help/submit_tex.md

```diff
@@ -25,14 +25,18 @@ We will soon be rolling out changes to how arXiv process (La)TeX submissions. Th
 
  1. The AutoTeX system would try different versions of TeX to see which one successfully builds a PDF. Going forward we will only use the version of TeX currently used by arXiv.
     - Our plan is that arXiv's "current" version closely track the annual TeX Live releases. In the past, we have often gone a few years between TeX updates.
- 1. The AutoTeX system would attempt to detect image files that were not referenced in the (La)TeX source for the paper and append those images to the end of the PDF for the paper. Based on recent feedback, we find that most authors are surprised by this behvior, and we will thus be discontinuing this practice. If an author wants images at the end of their paper, just use the normal TeX procedures for this (see https://latex-tutorial.com/tutorials/figures/).
+ 1. The AutoTeX system would attempt to determine which files in a submission were part of the main document, and which weren't. We're no longer going to do this.
+      - If there were multiple files ending in `.tex`, it would create PDFs for those extra files and append them to main paper's PDF. Now, if authors want a .tex file to be part of their main paper, they should use \include or \input commands to include the file. See https://www.baeldung.com/cs/latex-include-vs-input.
+      - If authors need the previous apppend behavior for a submission with multiple `.tex` files, please follow the instructions here: https://info.arxiv.org/help/00README.html.
+      - If image files (JPG, PNG, PDF) files were found, they would be rendered and appended verbatim to the main paper's PDF. If authors want to include images anywhere in their paper, they should use the normal TeX contructs for this. See https://latex-tutorial.com/tutorials/figures/. 
+  
   1. The AutoTeX system would pre-load the LaTeX hyperref package to LaTeX submissions that do not already include it. This package add changes various references to active links in the PDF document (eg, clicking on a reference jumps to the entry in the bibliography section). We will no longer automatically try to add this package. (Note – before you add a \usepackage{hyperref} to your main TeX file, check to see if the template you are using already has a reference by checking the PDF for clickable links; most do).
-1. The AutoTeX system would look for any references in a paper that looked like an arXiv paper ID (such as 	arXiv:2402.08954, or 2402.08954), and turn the into a hyperlink to the paper on arXiv. 
+ 1. The AutoTeX system would look for any references in a paper that looked like an arXiv paper ID (such as 	arXiv:2402.08954, or 2402.08954), and turn the into a hyperlink to the paper on arXiv. 
```

Comment:
1. The AutoTeX system would look for any references in a paper that looked like an arXiv paper ID (such as arXiv:2402.08954, or 2402.08954), and turn the ID into a hyperlink to the paper on arXiv.